### DISCUSSION TITLE
brew repo check for Security?

### DISCUSSION THREAD

**Discussion by simpsonetti** [Homebrew/discussions#4208] [Getting started]
### Output of `brew config`

```shell
what to do?
```

### Output of `brew doctor`

```shell
here also
```

### Description of issue

Hi, some days ago I installed "brew install izip" . 
Than executing izip doesn't show me that program. Hmm, then some time later, I deinstalled it. 
Next day my Security App shows me his summary and there was one entry from the day before: izip was in quarantine. Sentinel blocks it from starting. So my question is:
is the brew repository checked for security? Are the programs that can be installed on my MacBook checked for backdoors, virus, ransomeware and so on. 

If not, this is a major securtyfail and should be changed as soon as possible.

Are there some statements?
Regards Sebastian

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Acceptable-Casks#apps-that-bundle-malware
